src/pal/inc/pal.h

```diff
@@ -4086,13 +4076,8 @@ PAL_GetCurrentThreadAffinitySet(SIZE_T size, UINT_PTR* data);
 #define wcstok        PAL_wcstok
 #define wcscspn       PAL_wcscspn
 #define iswprint      PAL_iswprint
```

Comment:
Delete `iswprint` and `iswspace` here as well, and switch them to the same plan as the rest?